.storybook/main.js

```diff
@@ -1,5 +1,9 @@
 const { resolve } = require('path');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const {
+  twingEnvironmentPlugin,
+  twingOptions,
+} = require('../twing-environment-webpack-plugin');
```

Comment:
Since there are two files prefixed with `twing-`, do you think we should create a directory for these files?

Reply:
Sure